doc/runtime.rst

```diff
@@ -107,6 +107,15 @@ The functions for changing this default behavior are:
     The function ``Client.use_model_ensemble_prefix()`` controls
     object prefixing for model and script data.
 
+Model Execution Environment Variable
+====================================
+
+SmartRedis enables a timeout on the length of time that is allowed
```

Comment:
I might suggest leading with the variable name, i.e. `SR_MODEL_TIMEOUT` sets the amount of time that the client will wait for a model to execute [...]